lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -1420,24 +1444,20 @@ static bool isDifferentiationParameter(SILArgument *argument,
   return false;
 }
 
-/// For a nested function call that has results active on the differentiation
-/// path, compute the set of minimal indices for differentiating this function
-/// as required by the data flow.
+/// For an `apply` instruction with active results, compute:
+/// - The results of the `apply` instruction, in type order.
+/// - The set of minimal parameter and result indices for differentiating the
+///   `apply` instruction.
```

Comment:
Note: `collectMinimalIndicesForFunctionCall` now computes `SmallVectorImpl<SILValue> &results` instead of taking it as an operand. In practice, this does not change efficiency (results are computed in both cases) but simplifies call sites.